### PR TITLE
Refactor dates in feature attributes to be uniform

### PR DESCRIPTION
**Pull request by srezacova**
## Description

1. Refactor date format to be saved in feature attributes table to be uniform with dates uploaded from Azure functions
2. Rename region to feature
3. Add featureType that is saved as feature_id_lvl
4. Fix duplicates in feature attributes created by running initial load more times

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have checked to ensure that there aren't other open [Pull Requests](../../../pulls) for the same update/change.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: initial-data-load/src/types.ts [hunk 1]
```diff
@@ -38,9 +38,10 @@ export interface Metadata {
 }
 
 // API DATA FORMATS //
-export interface APIRegionAttribute {
+export interface APIFeatureAttribute {
   attributeId: string;
   featureId: string;
+  featureType: string;
```
Comment (bariela) [inline initial-data-load/src/types.ts]: I would make it optional, as we might want to load data without specified featureType

File: postgis_init/entrypoint_initdb/99_20220513000000_create_attributes_tables.sh [hunk 2]
```diff
@@ -33,7 +33,7 @@ psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-E
     created_at TIMESTAMP WITH TIME ZONE NULL DEFAULT NOW(),
     updated_at TIMESTAMP WITH TIME ZONE NULL DEFAULT NOW(),
 
-    CONSTRAINT "attribute_UQ" UNIQUE (attribute_id, feature_id, feature_id_lvl, date_ISO)
+    CONSTRAINT "attribute_without_feature_id_lvl_UQ" UNIQUE (attribute_id, feature_id, date_iso)
```
Comment (bariela) [inline postgis_init/entrypoint_initdb/99_20220513000000_create_attributes_tables.sh]: the previous constraint name sounds better, as we have only one

Comment (srezacova) [reply postgis_init/entrypoint_initdb/99_20220513000000_create_attributes_tables.sh]: Yeah sure, fixed.

File: initial-data-load/src/types.ts (hunk 1 above)
Comment (srezacova) [reply initial-data-load/src/types.ts]: Yeah sure, fixed.

**Pull request merged**